packages/utils/telemetry-utils/src/events.ts

```diff
@@ -8,9 +8,25 @@
 import { EventEmitter } from "events";
 import { ITelemetryLoggerExt } from "./telemetryTypes";
 
+/**
+ * Note: These contents really don't belong in this package.
+ * They should be moved into the `core-utils` package in the future.
+ */
```

Comment:
Nitpick: I assume you meant all the contents of this file. Might be worth a 1 line explanation of why